ferretdb/ferretdb.go

```diff
@@ -134,8 +134,10 @@ func New(config *Config) (*FerretDB, error) {
 
 		SQLiteURL: config.SQLiteURL,
 
+		//nolint:mnd // Handler default values
```

Comment:
Those default values are handled in kong tags, hence I believe there's no need of introducing constants to remove complaints about magic numbers.

Reply:
> Those default values are handled in kong tags

So, they are not handler default values. The code comment is therefore incorrect